src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureArea,DrawFIgure {
```

Comment:
Don't use abstract classes to set behavior for classes. In this case, it seems like you are using an abstract class to provide common behavior for figures, but consider if an interface or a regular class would be more appropriate depending on the shared characteristics and behaviors of the figures.